raphtory/src/db/api/view/internal/into_dynamic.rs

```diff
@@ -37,23 +37,23 @@ impl<G: GraphViewOps> IntoDynamic for VertexSubgraph<G> {
 macro_rules! graph_tag {
     () => {
         pub trait GraphTag:
-            $crate::db::view_api::GraphViewOps + $crate::db::view_api::into_dynamic::IntoDynamic
+            $crate::db::view::GraphViewOps + $crate::db::view::into_dynamic::IntoDynamic
         {
         }
 
         impl GraphTag for $crate::db::graph::Graph {}
-        impl<G: $crate::db::view_api::GraphViewOps> GraphTag
+        impl<G: $crate::db::view::GraphViewOps> GraphTag
             for $crate::db::graph_layer::LayeredGraph<G>
         {
         }
-        impl<G: $crate::db::view_api::GraphViewOps> GraphTag
+        impl<G: $crate::db::view::GraphViewOps> GraphTag
             for $crate::db::graph_window::WindowedGraph<G>
         {
         }
-        impl<G: $crate::db::view_api::GraphViewOps> GraphTag
+        impl<G: $crate::db::view::GraphViewOps> GraphTag
             for $crate::db::subgraph_vertex::VertexSubgraph<G>
         {
         }
-        impl GraphTag for $crate::db::view_api::internal::DynamicGraph {}
+        impl GraphTag for $crate::db::view::internal::DynamicGraph {}
     };
 }
```

Comment:
I added this but I think it shouldn't be used anymore? It is not useful now that python is in the core crate...

Reply:
Removed